### PR TITLE
[excel] (Requirement Set) Custom Functions requirement set information

### PR DESCRIPTION
**Pull request by AlexJerabek**
Staging this PR for when custom functions are ready roll out in prod.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[8d22f8c](https://github.com/OfficeDev/office-js-docs-pr/pull/919/commits/8d22f8c460de404a71cfa96f332b9e4d7452493c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/overview/office-add-in-availability.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-CFReqSet/docs/overview/office-add-in-availability.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/excel-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-CFReqSet/docs/reference/requirement-sets/excel-api-requirement-sets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C24%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904242351574385-919%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=N81tpvQn03GnG780cOialT6F9WFFjP8UXo%2B3RLdQCgY%3D&st=2019-04-24T23%3A48%3A08Z&se=2019-05-25T23%3A53%3A08Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/requirement-sets/excel-api-requirement-sets.md [hunk 1]
```diff
@@ -32,6 +32,12 @@ Excel add-ins run across multiple versions of Office, including Office 2016 or l
 > [!NOTE]
 > The build number for Office 2016 installed via MSI is 16.0.4266.1001. This version only contains the ExcelApi 1.1 requirement set.
 
+[Custom Functions](../../excel/custom-functions-overview.md) use separate requirement sets from the core Excel JavaScript APIs. The following table lists the Custom Functions requirement sets, the supported Office host applications, and the build versions or number for those applications.
+
+|  Requirement set  |  Office 365 for Windows  |  Office 365 for iPad  |  Office 365 for Mac  | Office Online  |
+|:-----|-----|:-----|:-----|:-----|
+| CustomFunctionsRuntime 1.1 | Version 1904 (Build 11601.20072) or later | Not supported | 16.24 or later | April 2019 | Coming soon |
```
Comment (mscharlock) [inline docs/reference/requirement-sets/excel-api-requirement-sets.md]: I'm assuming these are right but you might want to double check with Keyur in person just to triple make sure

**Comment by o365devx** [comment]
Docs Build status updates of commit _[00d384a](https://github.com/OfficeDev/office-js-docs-pr/pull/919/commits/00d384a8fd2eb01fbcc61c2c45bc3d7cfaefbb9a)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-6b38ccce)

<a id="6b38ccce"></a>
###
 - **[Error]** Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/OfficeDev/office-js-docs-pr, working directory: W:\q1ea-s, target branch: master, target branch commit: d37503f74d89384b8c1368d1f2f443ed7edd8e26, source commit: 00d384a8fd2eb01fbcc61c2c45bc3d7cfaefbb9a.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904291727482798-919%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=KDjXMdVCeLxRx2rsYD8gh2FHY0yddltqd1IjOp8WG0c%3D&st=2019-04-29T17%3A22%3A59Z&se=2019-05-30T17%3A27%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/requirement-sets/excel-api-requirement-sets.md (hunk 1 above)
Comment (AlexJerabek) [reply docs/reference/requirement-sets/excel-api-requirement-sets.md]: Confirmed with Keyur internally.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[3150326](https://github.com/OfficeDev/office-js-docs-pr/pull/919/commits/3150326672362bf7aebcc734580003948015e0c0)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/overview/office-add-in-availability.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-CFReqSet/docs/overview/office-add-in-availability.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/excel-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-CFReqSet/docs/reference/requirement-sets/excel-api-requirement-sets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904291731541119-919%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=s0HC%2BvMer5ZepHv0VgQlVwhB7y3qtw5%2Bptld5DSgjvg%3D&st=2019-04-29T17%3A28%3A07Z&se=2019-05-30T17%3A33%3A07Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by AlexJerabek** [comment]
Thanks for the review @mscharlock! Updated for your approval.

**Comment by mscharlock** [comment]
Looks good! Thanks! :)

**Comment by AlexJerabek** [comment]
@mscharlock Cool. I'll mark as holding. Please check in when CFs go to prod.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[d7cb3ba](https://github.com/OfficeDev/office-js-docs-pr/pull/919/commits/d7cb3ba0f9fdbf6ea953bc9095cc72030d29b8dc)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-587b071a)

<a id="587b071a"></a>
###
 - **[Error]** Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/OfficeDev/office-js-docs-pr, working directory: W:\s01v-s, target branch: master, target branch commit: 5190da5499f7a04d47521de7eee1d3626190dd0b, source commit: d7cb3ba0f9fdbf6ea953bc9095cc72030d29b8dc.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C1%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905012008390656-919%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=E%2FSldJtyM4Aqz8mx9QhyWW1dEwCx8EwuWoHDNnqEIkM%3D&st=2019-05-01T20%3A03%3A50Z&se=2019-06-01T20%3A08%3A50Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[361dfb5](https://github.com/OfficeDev/office-js-docs-pr/pull/919/commits/361dfb59f4a53040da391282be7883b02f9311ef)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/office-versions-and-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-CFReqSet/docs/develop/office-versions-and-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/overview/office-add-in-availability.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-CFReqSet/docs/overview/office-add-in-availability.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/excel-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-CFReqSet/docs/reference/requirement-sets/excel-api-requirement-sets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C1%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905012012431216-919%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=1MojPWiYqMyJT0XOXvaHqYHMMAx4u%2BYXPEzAK0eW8u0%3D&st=2019-05-01T20%3A08%3A56Z&se=2019-06-01T20%3A13%3A56Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/develop/office-versions-and-requirement-sets.md [hunk 2]
```diff
@@ -32,7 +31,7 @@ In addition, other functionality such as add-in commands (ribbon extensibility)
 
 An add-in can only use APIs in requirement sets that are supported by the version of Office host where the add-in is running. To know exactly which requirement sets are available for a specific Office host version, refer to the following host-specific requirement set articles:
 
-- [Excel JavaScript API requirement sets](/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets) (ExcelApi)
+- [Excel JavaScript API requirement sets](/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets) (ExcelApi, [CustomFunctionsRuntime](../excel/custom-functions-overview.md))
```
Comment (davidchesnut) [inline docs/develop/office-versions-and-requirement-sets.md]: The link to custom functions overview is odd. This article has specific info about the runtime https://docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-architecture?view=office-js Link to this instead?

File: docs/overview/office-add-in-availability.md [hunk 3]
```diff
@@ -37,6 +38,7 @@ To work as expected, your Office Add-in might depend on a specific Office host,
         - <a href="/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets">ExcelApi 1.7</a><br>
         - <a href="/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets">ExcelApi 1.8</a><br>
         - <a href="/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets">ExcelApi 1.9</a><br>
+        - <a href="/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets">CustomFunctionsRuntime 1.1</a><br>
```
Comment (davidchesnut) [inline docs/overview/office-add-in-availability.md]: Would be nice to deep link this. When I clicked and saw top of the page I was like "where's custom functions"? Is there a way to give it an H2 so you can deep link to the custom functions section?

File: docs/overview/office-add-in-availability.md [hunk 4]
```diff
@@ -165,6 +170,7 @@ To work as expected, your Office Add-in might depend on a specific Office host,
...
```
Comment (davidchesnut) [inline docs/overview/office-add-in-availability.md]: I feel like on iPad support to double-check with Keyur just to be sure.

File: docs/reference/requirement-sets/excel-api-requirement-sets.md [hunk 5]
```diff
@@ -33,6 +33,12 @@ Excel add-ins run across multiple versions of Office, including Office 2016 or l
 > [!NOTE]
 > The build number for Office 2016 installed via MSI is 16.0.4266.1001. This version only contains the ExcelApi 1.1 requirement set.
 
+[Custom Functions](../../excel/custom-functions-overview.md) use separate requirement sets from the core Excel JavaScript APIs. The following table lists the Custom Functions requirement sets, the supported Office host applications, and the build versions or number for those applications.
+
+|  Requirement set  |  Office 365 for Windows  |  Office 365 for iPad  |  Office 365 for Mac  | Office Online  |
```
Comment (davidchesnut) [inline docs/reference/requirement-sets/excel-api-requirement-sets.md]: The last table header should read "Office Online Server" I assume. Also would check with Keyur if that is planned to be supported.

File: docs/reference/requirement-sets/excel-api-requirement-sets.md [hunk 6]
```diff
@@ -33,6 +33,12 @@ Excel add-ins run across multiple versions of Office, including Office 2016 or l
 > [!NOTE]
 > The build number for Office 2016 installed via MSI is 16.0.4266.1001. This version only contains the ExcelApi 1.1 requirement set.
 
+[Custom Functions](../../excel/custom-functions-overview.md) use separate requirement sets from the core Excel JavaScript APIs. The following table lists the Custom Functions requirement sets, the supported Office host applications, and the build versions or number for those applications.
```
Comment (davidchesnut) [inline docs/reference/requirement-sets/excel-api-requirement-sets.md]: Make this an H2 for deep links?

**Comment by o365devx** [comment]
Docs Build status updates of commit _[a80e006](https://github.com/OfficeDev/office-js-docs-pr/pull/919/commits/a80e0062399d30d28f1666899d9273b3136209de)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/office-versions-and-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-CFReqSet/docs/develop/office-versions-and-requirement-sets.md) | :white_check_mark:Succeeded | |
[docs/overview/office-add-in-availability.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-CFReqSet/docs/overview/office-add-in-availability.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/excel-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-CFReqSet/docs/reference/requirement-sets/excel-api-requirement-sets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C1%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905012038069760-919%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=fYOFWCSW7uuOznsnHLawomFPQCAw0UFKkBsICeY%2FCYA%3D&st=2019-05-01T20%3A34%3A21Z&se=2019-06-01T20%3A39%3A21Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by AlexJerabek** [comment]
Thanks @davidchesnut. Updated as per feedback. I'm not really sure how to format [this section](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/office-add-in-availability?branch=AlexJ-CFReqSet#custom-functions). Any thoughts are welcome. We can also get this in before the bash and circle back later.

**Pull request merged**